pmm-app/tests/QAN/filters_test.js

```diff
@@ -203,6 +209,83 @@ Scenario(
     qanFilters.applyFilter(serviceName);
     const percentageAfter = await qanFilters.getPercentage('Service Type', serviceType);
 
-    assert.ok(percentageAfter !== percentageBefore, 'Percentage for filter Service Type was expected to change');
+    assert.ok(
+      percentageAfter !== percentageBefore,
+      'Percentage for filter Service Type was expected to change',
+    );
+  },
+);
+
+Scenario(
+  'PMM-T436 - Verify short-cut navigation from filters to related dashboards - Cluster @qan',
+  async (I, qanFilters, dashboardPage) => {
+    const mongoLink = '/graph/d/mongodb-cluster-summary/mongodb-cluster-summary';
+    const header = 'MongoDB Cluster Summary';
+
+    I.fillField(qanFilters.fields.filterBy, 'mongodb');
+    qanFilters.navigateByShortCut(mongoLink);
+    // wait for open new tab
+    I.wait(2);
```

Comment:
This can be replaced with `waitAndSwitchTabs` which is available in dashboardPage, please check once, it checks for tab counts to increase then switches.

Reply:
I added the `waitAndSwitchTabs` but still must have `I.wait(2)` because it's failing without it.